firefox_ui_tests/remote/security/test_dv_certificate.py

```diff
@@ -0,0 +1,103 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import By, Wait
+
+from firefox_ui_harness.decorators import skip_under_xvfb
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+class TestDVCertificate(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+        self.test_url = "https://ssl-dv.mozqa.com"
+        self.cert = None
```

Comment:
No need to define it at this level. We only had to do it in Mozmill Tests because of the callbacks. Lets make it a normal variable in the test.